worker/worker.go

```diff
@@ -33,15 +33,18 @@ import (
 )
 
 const (
-	lockingPriorityRenew      = 100 // highest
-	lockingPriorityPriceTable = 95
-	lockingPriorityFunding    = 90
-	lockingPrioritySyncing    = 80
-
-	lockingDurationRenew      = time.Minute
-	lockingDurationPriceTable = 30 * time.Second
-	lockingDurationFunding    = 30 * time.Second
-	lockingDurationSyncing    = 30 * time.Second
+	lockingPriorityActiveContractRevision = 105 // highest
+	lockingPriorityRenew                  = 100
+	lockingPriorityPriceTable             = 95
+	lockingPriorityFunding                = 90
+	lockingPrioritySyncing                = 80
+	lockingPriorityUpload                 = 1 // lowest
+
+	lockingDurationActiveContractRevision = time.Minute
```

Comment:
I think this should be 0? It's kind of meaningless anyway because currently fetching let's say 150 active contracts takes > 1 minute anyway so by the time you're done fetching all of them the first locks will have expired already.

But most importantly: the contractor only uses the revisions for contract checks and will reacquire the lock in case it wants to renew or refresh the contract.

Reply:
You might have misread. If we fetch 150 contracts, we also lock 150 contracts with `lockingDurationActiveContractRevision` for each one. 

Fetching the revision revises the contract so we have to lock the contract and can't pass 0. Regardless if we ever were allowed to pass 0, we wouldn't need to acquire the contract in the first place.